cmd/juju/model/showgeneration.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package model
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+
+	"github.com/juju/juju/api/modelgeneration"
+	jujucmd "github.com/juju/juju/cmd"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/cmd/output"
+	"github.com/juju/juju/core/model"
+)
+
+const (
+	showGenerationSummary = `Displays a summary of the active "next" generation.`
+	showGenerationDoc     = `
+Summary information includes each application that has changes made under the
+generation, with any units that have been advanced, and any changed config
+values made to the generations.
+
+Examples:
+    juju add-generation
+
+See also:
+	add-generation
```

Comment:
Should this be left-aligned like the ones below it?

Reply:
Yes; tabs vs spaces. Corrected.